packages/emails/email-manager.ts

```diff
@@ -110,60 +114,70 @@ export const sendScheduledEmails = async (
 
 // for rescheduled round robin booking that assigned new members
 export const sendRoundRobinScheduledEmails = async (calEvent: CalendarEvent, members: Person[]) => {
+  const calendarEvent = formatCalEvent(calEvent);
```

Comment:
NIT: for consistency rename "const calendarEvent" to "const formattedCalEvent" to match other places.